wrappers/services.go

```diff
@@ -88,6 +88,8 @@ Before=slices.target
 # Always enable memory accounting otherwise the MemoryMax setting does nothing.
 MemoryAccounting=true
 MemoryMax=%[2]d
+# for compatibility with older versions of systemd
+MemoryLimit=%[2]d
```

Comment:
Does systemd complain a lot when both are present?

Reply:
Not as far as I can tell, old systemd's just complains when the new one is there, but we have other such things like the `LazyUnmount` setting that are like this anyways generating messages. Some customers have complained about these messages (unit setting not supported or some such for new settings on old systemds) however so in an ideal world we would not write out the new ones, but I think it's fine for now.